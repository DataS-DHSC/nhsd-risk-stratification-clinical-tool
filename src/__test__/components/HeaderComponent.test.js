import React from 'react';
import { render } from '@testing-library/react';
import HeaderComponent from '../../components/HeaderComponent';

const setup = () => render(<HeaderComponent />);

describe('HeaderComponent', () => {
  it('should match snapshot ', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
