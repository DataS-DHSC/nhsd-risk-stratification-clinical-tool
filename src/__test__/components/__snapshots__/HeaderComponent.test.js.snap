// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent should match snapshot  1`] = `
<DocumentFragment>
  <header
    class="nhsuk-header"
    id="nhsuk-header"
    role="banner"
  >
    <div
      class="nhsuk-width-container nhsuk-header__container"
    >
      <div
        class="nhsuk-header__logo nhsuk-header__logo--only"
      >
        <a
          aria-label="NHS homepage"
          class="nhsuk-header nhsuk-header__link--service"
          id="header-home-link"
        >
          <svg
            class="nhsuk-logo"
            focusable="false"
            role="presentation"
            viewBox="0 0 40 16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="nhsuk-logo__background"
              d="M0 0h40v16H0z"
            />
            <path
              class="nhsuk-logo__text"
              d="M3.9 1.5h4.4l2.6 9h.1l1.8-9h3.3l-2.8 13H9l-2.7-9h-.1l-1.8 9H1.1M17.3 1.5h3.6l-1 4.9h4L25 1.5h3.5l-2.7 13h-3.5l1.1-5.6h-4.1l-1.2 5.6h-3.4M37.7 4.4c-.7-.3-1.6-.6-2.9-.6-1.4 0-2.5.2-2.5 1.3 0 1.8 5.1 1.2 5.1 5.1 0 3.6-3.3 4.5-6.4 4.5-1.3 0-2.9-.3-4-.7l.8-2.7c.7.4 2.1.7 3.2.7s2.8-.2 2.8-1.5c0-2.1-5.1-1.3-5.1-5 0-3.4 2.9-4.4 5.8-4.4 1.6 0 3.1.2 4 .6"
            />
            <image
              alt="NHS Logo"
              src="https://assets.nhs.uk/images/nhs-logo.png"
              xlink:href=""
            />
          </svg>
          <span
            class="nhsuk-header__service-name"
          >
            COVID-19 Clinical Risk Assessment Tool
          </span>
        </a>
      </div>
    </div>
    <div
      class="top-banner"
    >
      <div
        class="nhsuk-width-container"
        id="beta-warning-banner"
      >
        <strong
          class="status-box status-box-blue"
        >
          BETA
        </strong>
        <span>
           We want to hear from you. Give us 
          <a
            href="/ras/page/feedback"
            id="beta-warning-banner-link"
            rel="noreferrer noopener nofollow"
            target="_blank"
          >
            feedback
          </a>
           to help us improve this service.
        </span>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
